packs/spells/falling-stars.json

```diff
@@ -118,13 +131,20 @@
                 "system": {
                     "damage": {
                         "value": {
-                            "i7p7fmqd0A8a64Oz": {
-                                "applyMod": false,
-                                "type": {
-                                    "subtype": "",
-                                    "value": "cold"
+                            "value": {
+                                "5hgRJg7CfV9pZSUa": {
+                                    "type": {
+                                        "value": "cold"
+                                    }
```

Comment:
This looks malformed, it would lead to a `value.value`

Reply:
You're right, fixed that. Thanks for double checking!